DMCompiler/DM/Visitors/DMVisitorExpression.cs

```diff
@@ -84,6 +84,11 @@ class DMVisitorExpression : DMASTVisitor {
             } else if (name == "args") {
                 Result = new Expressions.Args();
             } else {
+                if (_proc is null)
```

Comment:
What code causes this to happen?

Reply:
Setting an unknown identifier to an unknown identifier in an object definition.